Documentation/Helm-Charts/operator-chart.md

```diff
@@ -56,6 +56,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.attacher.image` | Kubernetes CSI Attacher image | `registry.k8s.io/sig-storage/csi-attacher:v4.1.0` |
 | `csi.cephFSFSGroupPolicy` | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted. supported values are documented at https://kubernetes-csi.github.io/docs/support-fsgroup.html | `"File"` |
 | `csi.cephFSPluginUpdateStrategy` | CSI CephFS plugin daemonset update strategy, supported values are OnDelete and RollingUpdate | `RollingUpdate` |
+| `csi.cephFSPluginUpdateStrategyMaxUnavailable` | A maxUnavailable parameter of CSI CephFS plugin daemonset update strategy. | `1` |
```

Comment:
If you need to edit this file, 
You also need to edit the `Edit the corresponding *.gotmpl.md file instead
`

Reply:
I restored the file.
Thank you.